### DISCUSSION TITLE
Reuse space after island disband to reduce world filesize

### DISCUSSION THREAD

**Discussion by Baterka** [BG-Software-LLC/SuperiorSkyblock2#211] [Denied Feature Requests] [answered]
Plugin clears chunks after island is disbanded by player, but empty space is not reused aftewards leaving behind ~2MB region each time. I was hoping that SS2 will have this feature, because I was using BentoBox and my world had 100GB after a while.

**Comment by OmerBenGera** [comment, marked as answer]
I am denying this as SuperiorSkyblock generates islands in a circular pattern. I don't believe you'll get to a 100GB world.
If it occurs, open an issue as it's more of an unintended behavior than a feature-request.
